docker-compose.yml

```diff
@@ -146,6 +146,17 @@ services:
       mode: replicated
       replicas: 1
 
+  dsv:
```

Comment:
Hmm..`dso` or `dsv`?

Reply:
Yeah... Perhaps it should be `dso` already. I've already made a ticket for the rename. Engine is using `dsp` currently, the repository was named `diagnosis-specific-view`, and the actual name would be `dso`.

Reply:
https://jira.duodecim.fi/browse/EBMEDS-1309 Unify naming scheme for DSO ticket created.